src/controllers/users.controller.js

```diff
@@ -0,0 +1,81 @@
+const { usersService } = require('./../services');
+
+const getAll = (req, res) => {
+  const users = usersService.getAll();
+
+  res.json(users);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const user = usersService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.json(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = usersService.create(name);
+
+  res.status(201).json(newUser);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
```

Comment:
Similar to the `create` function, consider adding validation for the `name` field in the `update` function to ensure it meets any required criteria.